yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/domain/services/StatisticsService.java

```diff
@@ -11,4 +13,6 @@ public interface StatisticsService {
     ArrayList<PerTeammateStatisticsEntity> getPerTeammateStatistics(StatisticsRequestEntity requestEntity);
     ArrayList<PerTeamStatisticsEntity> getPerTeamStatistics(StatisticsRequestEntity requestEntity);
     ArrayList<PerWeekdayStatisticsEntity> getPerWeekDayStatistics(StatisticsRequestEntity requestEntity);
+    List<LastActivityEntity> getLastTeamsActivityByCustomerId (Integer requestEntity);
```

Comment:
requestEntity as the name of the parameter is not very clear i think maybe customerId and TeamId would be better